test/testfunctions.cpp

```diff
@@ -1352,6 +1359,33 @@ class TestFunctions : public TestFixture {
               "}");
         ASSERT_EQUALS("[test.cpp:4]: (portability) The 2nd memset() argument '1.0f+i' is a float, its representation is implementation defined.\n", errout.str());
     }
+
+    // NRVO check
+    void returnLocalStdMove1() {
+        check("struct A{}; A f() { A var; return std::move(var); }");
```

Comment:
You should also have a check for `struct A{}; A f(A var) { return std::move(var); }` as well(as arguments are implicitly moved).